### PR TITLE
Fix calculate_subtask_verification_time to use only concent settings

### PR DESCRIPTION
**Pull request by pawelkisielewicz**
Resolves https://github.com/golemfactory/concent/issues/628

### CODE REVIEW

File: concent_api/concent_api/settings/testing.py [hunk 1]
```diff
@@ -60,14 +60,14 @@
...
 
-CONCENT_MESSAGING_TIME    = 2
-
 STORAGE_SERVER_INTERNAL_ADDRESS = 'http://localhost/'
 
 VERIFIER_STORAGE_PATH = '/tmp/'
 
 CONCENT_MESSAGING_TIME    = 0
 
+CONCENT_UPLOAD_RATE       = int(384 / 8)  # KB/s = kbps / 8
```
Comment (pawelkisielewicz) [inline concent_api/concent_api/settings/testing.py]: Default value the same as in golem constants

File: concent_api/core/tests/utils.py [hunk 2]
```diff
@@ -167,7 +168,7 @@ def _get_deserialized_report_computed_task(
 
     def _get_deserialized_task_to_compute(
         self,
-        timestamp: Optional[str] = None,
+        timestamp: Optional[Union[str, datetime.date]] = None,
```
Comment (dybi) [inline concent_api/core/tests/utils.py]: isn't it `datetime.datetime`?, besides, I think `Union[str, datetime.datetime, None]` would be more readable ;)

File: concent_api/concent_api/settings/testing.py (hunk 1 above)
Comment (dybi) [reply concent_api/concent_api/settings/testing.py]: what about `development.py`?

File: concent_api/concent_api/settings/testing.py [hunk 3]
```diff
@@ -60,14 +60,14 @@
 
 STORAGE_CLUSTER_ADDRESS = 'http://localhost/'
 
-CONCENT_MESSAGING_TIME    = 2
```
Comment (rwrzesien) [inline concent_api/concent_api/settings/testing.py]: Why are you removing this ?

Comment (kbeker) [reply concent_api/concent_api/settings/testing.py]: This should be still in our testing settings. It doesn't matter that we overriding settings in our tests (at least we should) but in `testing.py` we must have all settings what we have in `base.py`

Comment (pawelkisielewicz) [reply concent_api/concent_api/settings/testing.py]: It was duplicated. look at line 69 - there is ` CONCENT_MESSAGING_TIME    = 0`

File: concent_api/concent_api/settings/testing.py (hunk 1 above)
Comment (pawelkisielewicz) [reply concent_api/concent_api/settings/testing.py]: done

File: concent_api/core/tests/utils.py (hunk 2 above)
Comment (pawelkisielewicz) [reply concent_api/core/tests/utils.py]: done

File: concent_api/concent_api/settings/testing.py (hunk 3 above)
Comment (kbeker) [reply concent_api/concent_api/settings/testing.py]: So this should stay and delete this with value 0.

**Pull request merged**